components/Profile/Settings.js

```diff
@@ -80,17 +98,9 @@ const Settings = ({ user, isEditing, onChange, values, errors, dirty, t }) => {
     </Checkbox>
     <br />
```

Comment:
```suggestion
     <br style={{ clear: 'left' }} />
```
_preexisting condition but let's fix it_